docs/knowledgebase/learn-substrate/off-chain-workers.md

```diff
@@ -25,7 +25,7 @@ easily access on-chain state for their computations.
 
 ## APIs
 
-Offchain workers have access to extended APIs for communicating with the external world:
+OCWs have access to extended APIs for communicating with the external world:
```

Comment:
Even though we have introduced the acronym, I still think it's best in this doc to _always_ use the whole name written out. Someone may not necessarily read the doc from top to bottom.
```suggestion
Off-chain workers have access to extended APIs for communicating with the external world:
```